### DISCUSSION TITLE
[Problem] PR introduction text is being ignored

### DISCUSSION THREAD

**Discussion by berniev** [FreeCAD/FreeCAD#10195] [Discussions that will turn in to issues but need refining]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

0.20 (Release)

### Full version info

```shell
na
```

### Subproject(s) affected?

None

### Problem description

Further to #8657 the current structure of PR introduction is being ignored or deleted in many cases, including by senior members of this project.

The checkmark (place an 'x' to make a tick [huh?]) is counterintuitive and doesn't work if a leading or trailing space. An interesting UI design failure.

And who's going to read all that text, except maybe a first-timer. Maybe.

With no reward for jumping through these hoops and no punishment for not jumping through them, it is only natural they will be ignored, and possibly resented as cognitive distractions.

IMHO the whole text should be reduced to a couple of succinct, polite, lines and a couple of links.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
Please add a legit title to this ticket l, TIA

**Comment by berniev** [comment]
Oops! Tks!

**Comment by adrianinsaval** [comment]
why create a separate issue for this and not jsut add this as comment in that issue?

**Comment by berniev** [comment]
Because this report is addresses the **fact** that lots of PRs are ignoring the introduction, and attempting to identify the cause and a possible solution, not just expressing a thought.

**Comment by benj5378** [comment]
This should be a forum discussion and not an issue on GitHub. @berniev, please start a new thread on the forum with proposal of a new text/solution.

**Comment by berniev** [comment]
No, the policy of flicking issues back to forum was abandoned months ago.

**Comment by luzpaz** [comment]
I'm trying to convert this issue to a discussion and there is a problem. We need to do it manually.

**Comment by sliptonic** [comment]
The text is pretty low-value.  Looking closer:

> Thank you for creating a pull request to contribute to FreeCAD! 

It's nice that we're appreciative but people probably don't actually feel thanked by reading this.  It's boilerplate and can be removed.

>Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the >[FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.
>
>- [ ]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Checking this doesn't actually 'do' anything.  If it is checked, it doesn't mean the PR _actually_ satisfies requirements and if it isn't checked it does't mean it _doesn't_.   IMHO,  This could be replaced with a link to the CONTRIBUTING.md file and a warning that improperly submitted PRs may be cancelled or delayed.


>Please remember to update the Wiki with the features added or changed once this PR is merged.  
>**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

It seems premature to tell the contributor to update the wiki at this time.   It would be better if the information could be emailed to them when the PR is merged.

**Comment by adrianinsaval** [comment]
since the text has been removed I think we can close this
